lib/commands/deploynodeapp.js

```diff
@@ -7,7 +7,11 @@ var async = require('async');
 var fs = require('fs');
 var mustache = require('mustache');
 var _ = require('underscore');
+var post = require('request').post;
 var unzip = require('node-unzip-2');
+var targz = require('tar.gz')({}, {
```

Comment:
They will never see the temporary archive, so I don't think it matters, personally.